pkg/corerp/renderers/gateway/render_test.go

```diff
@@ -1499,6 +1499,7 @@ func validateContourHTTPRoute(t *testing.T, outputResources []rpv1.OutputResourc
 					},
 				},
 				PathRewritePolicy: expectedRewrite,
+				EnableWebsockets:  false,
```

Comment:
Should we also add a unit test for scenario where `route.EnableWebsockets` is `true`?

Reply:
I added a test by refactoring this file a little bit. We may need a better refactor in future.